docs/api.md

```diff
@@ -117,12 +119,45 @@ let result = vat.validate({
 // result.value.name === 'Ham Burger';
 ```
 
+### `any.clone()`
+Create a clone of the current schema that can be extended.
```

Comment:
Could maybe make this a bit more explicit, something like: "Create a clone of the current schema. Combine with `extend` to produce a modified schema that will not affect usage elsewhere."